mti_nma/steps/avgshape/avgshape.py

```diff
@@ -81,11 +81,12 @@ def run(self, sh_df=None, struct="Nuc", **kwargs):
 
         coeffs_avg = coeffs_avg.reshape(-2, lmax, lmax)
 
-        mesh_avg, _ = shtools.get_reconstruction_from_coeffs(coeffs=coeffs_avg)
+        # Here we use the new meshing implementation for a more evenly distributed mesh
+        mesh_avg, _ = shtools.get_even_reconstruction_from_coeffs(coeffs=coeffs_avg, npoints=1024)
 
         shtools.save_polydata(
             mesh=mesh_avg,
-            filename=str(avg_data_dir / f"avgshape_{struct}.vtk")
+            filename=str(avg_data_dir / f"avgshape_{struct}.ply")
```

Comment:
I would assume the `{struct}` stuff should be removed.

Reply:
Why? I realize its no longer **necessary** but I don't see it as a bad thing. Can you explain?

Reply:
Oh wait. This isn't the manifest. I thought all of these were the manifest for some reason. My bad.